app/views/child_benefit_tax/main.html+new_navigation.erb

```diff
@@ -0,0 +1,138 @@
+<% content_for :title, "Child Benefit tax calculator" %>
+<% content_for :body_classes, "full-width" %>
+
+<div class="grid-row">
+  <main id="content" role="main" class="group calculator-page">
```

Comment:
Is there a chance we could re-use more of the body of the page by moving that into a common partial? That would avoid content changes getting out of sync between versions.